tasks/config/run.js

```diff
@@ -177,8 +187,9 @@ module.exports = function (grunt) {
         ready: /Optimization .+ complete/,
         quiet: true
       },
-      cmd: buildScript,
+      cmd: binScript,
```

Comment:
Are you sure this change is alright?

Reply:
Whoops I forgot to comment on this.  @spalger were you concerned about a different node version being used?  I believe we're using the bin script anyways, the node build task isn't copied until after optimize.

I'll give this a double check.

Reply:
This is producing builds that don't have optimized assets because it's running the optimization in the repo, not the build output.